python-avd/pyavd/_eos_designs/structured_config/network_services/ip_igmp_snooping.py

```diff
@@ -114,6 +114,14 @@ def _ip_igmp_snooping_vlan(self: AvdStructuredConfigNetworkServices, vlan, tenan
             if version is not None:
                 ip_igmp_snooping_vlan["querier"]["version"] = version
 
+            fast_leave = default(
+                igmp_snooping_querier.get("fast_leave"),
```

Comment:
Move this condition out of the if condition on line 102. As condition on line 102 can be true even when evpn_l2_multicast is not enabled.

if evpn_l2_multicast_enabled:
            fast_leave = default(
                igmp_snooping_querier.get("fast_leave"),
                tenant_igmp_snooping_querier.get("fast_leave"),
                get(tenant, "evpn_l2_multicast.fast_leave"),
            )
            if fast_leave is not None:
                ip_igmp_snooping_vlan["fast_leave"] = fast_leave

Reply:
fixed!